config/settings/development.yml

```diff
@@ -9,8 +9,8 @@ betamocks:
   enabled: true
   recording: false
   # the cache dir depends on how you run the api
-  # cache_dir: ../vets-api-mockdata # via rails; e.g. bundle exec rails s or bundle exec rails c
-  cache_dir: /cache # via docker; e.g. make up or make console
+  cache_dir: ../vets-api-mockdata # via rails; e.g. bundle exec rails s or bundle exec rails c
+  # cache_dir: /cache # via docker; e.g. make up or make console
```

Comment:
was this something you had to do locally to get vets-api working in your setup? I'm not sure these changes should be part of the PR.

Reply:
yes, I was testing something. Will revert back to how it was!